Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -7381,7 +7381,7 @@ WebCodecsVideoEnabled:
     WebKitLegacy:
       default: false
     WebKit:
-      "PLATFORM(COCOA)": true
+      "USE(LIBWEBRTC)": WebKit::defaultPeerConnectionEnabledAvailable()
```

Comment:
Can you make this "USE(LIBWEBRTC) && !USE(GSTREAMER)" please?

Reply:
Done